docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -905,7 +905,16 @@ subquestion: |
   You still need to download and upload the package to your Docassemble
   playground to finish editing and refining it.
   
+  % if install_packages_api_key:
   ${ action_button_html(url_action('button_install_package'), label="Install your interview") }
+  % else:
+  **Sorry, you don't have an API key set up.** Ask your administrator to create
+  a Docassemble API key and save it to your configuration file, like this:
```

Comment:
I'd add a link here, in case it's a new admin who isn't quite familiar with things yet. 
https://docassemble.org/docs/api.html#manage_api